### DISCUSSION TITLE
GitHub Student - No ID to Upload (Online ASU Student)

### DISCUSSION THREAD

**Discussion by mccodav1** [community/community#30940] [GitHub Education]
Hello!

I'm trying to signup for the GitHub Student pack, but unfortunately, my application is requiring me to take a photo of a school ID.

Unfortunately, online ASU students do not _get_ a student ID!

I have access to the other documents it is suggesting, such as Portal access, transcripts, grades, schedules, syllabi, etc... but it says it will *only* accept ID pictures... of which I cannot obtain one.

Is there a workaround to this system? Anyone I can reach out to?

**Comment by Xrosguardian** [comment]
you can send a picture of your roll no. or the fee paid by you i did this and it worked

**Reply by mccodav1** [reply]
Where / who do I send it to?

**Reply by Xrosguardian** [reply]
When they ask for verification for being a student

**Reply by mccodav1** [reply]
The application itself requires a picture taken with my camera of the ID. There is no way for me to upload something, and nowhere else that asks me for this.

**Reply by Xrosguardian** [reply]
there is an option below to upload document tho.
Alternatively you could also print out the roll no. and then take its picture

**Reply by mccodav1** [reply]
I do appreciate your attempts to help, but I'm looking for another workaround here. There IS no option below for me to upload; the button is missing. Also, I have tried printer documents and taking a picture - the uploader tells me it is not a recognized ID, and that it must be an ID that is scanned.

**Reply by Xrosguardian** [reply]
are doing this on laptop or mobile?

**Reply by mccodav1** [reply]
I have tried on desktop, laptop and mobile.

**Reply by vineet-codes256** [reply]
> I have tried on desktop, laptop and mobile.

I'm facing the same issue. There is no Upload button, we have to take pictures with our laptop's camera in which it is impossible to get a clear image.
